src/pages/Confirm/Confirm.styled.jsx

```diff
@@ -53,6 +53,13 @@ export const CheckImg = styled.img`
     width: 147px;
   }
 
+  @media (min-width: 1023px) {
+    margin-left: auto;
+    position: relative;
+    left: 5vw;
+    width: 100px;
+  }
+
```

Comment:
This media query should be before of 1025px media query, just for hierarchy or "order"